pandas/_libs/tslib.pyx

```diff
@@ -105,7 +105,7 @@ from tslibs.timezones cimport (
     is_utc, is_tzlocal, is_fixed_offset,
     treat_tz_as_dateutil, treat_tz_as_pytz,
     get_timezone, get_utcoffset, maybe_get_tz,
-    get_dst_info
+    get_dst_info, _infer_dst
```

Comment:
let's give this a more descriptive name: ``infer_dst_transitions``

Reply:
sure